### DISCUSSION TITLE
How to debug 'unauthorized' responses?

### DISCUSSION THREAD

**Discussion by jhfoo** [umputun/remark42#1585] [Q&A]
The test page (/web) loads and the UI renders on a test page as described.  

I tried to enable anonymous comments via remark_auth_anon="true". After entering an anonymous username the page reloads with a broken image link and the Send button prints Not authorized in red.

It looks like the POST to /api/v1/comment returned (401 not authorized).

I'm testing on HTTPS on both the front end and hosting service. Also: testing on FreeBSD (service starts up fine). Let me know if you need more info.

**Comment by umputun** [comment]
pls, use discussion for such types of questions. moved

**Comment by angristan** [comment]
Did you find out why?
